aicoe/sesheta/review_manager.py

```diff
@@ -188,6 +188,10 @@ async def on_pr_open_or_edit(*, action, number, pull_request, repository, sender
                     data={"body": "This is an auto-approve of an auto-PR.", "event": "APPROVE"},
                 )
 
+                await github_api.post(
```

Comment:
strange... shouldnt the approving review be good enough?!

Reply:
It has to be an owner of the project or it could due to this `Prow will ignore most GitHub events generated by this account`, As stated [here](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#github-bot-account).